ipsec_offload_plugin/ipsec_offload/ipsec_offload.c

```diff
@@ -742,6 +777,20 @@ METHOD(kernel_ipsec_t, add_policy, status_t,
 			DBG2(DBG_KNL, "Inline_crypto_ipsec ipsec_tx_sa_classification_table:"
 			     "add entry failed");
 
+		err = vxlan_ipsec_tx_sa_classification_table(IPSEC_TABLE_ADD,
```

Comment:
It seems this table will allow subnet mask in ip addresses.  This is useful for tunnel mode only. If this table is used then the regular exact match fixed function table is not needed.
Instead of calling it vxlan* We can give a better name such as ipsec_tx_sa_classification_table_with_subnet().
This is only needed for tunnel mode. The exact match table will be used for transport mode. In this case we can reduce the no of arguments in this API !!